pages/Transfer.qml

```diff
@@ -532,84 +531,92 @@ Rectangle {
-
-            StandardButton {
-                id: signTxButton
-                text: qsTr("Sign tx file") + translationManager.emptyString
-                small: true
-                visible: !appWindow.viewOnly
-                onClicked: {
-                    console.log("Transfer: sign tx clicked")
-                    signTxDialog.open();
-                }
-            }
-
-            StandardButton {
-                id: submitTxButton
-                text: qsTr("Submit tx file") + translationManager.emptyString
-                small: true
-                visible: appWindow.viewOnly
-                enabled: pageRoot.enabled
-                onClicked: {
-                    console.log("Transfer: submit tx clicked")
-                    submitTxDialog.open();
-                }
-            }
-            
-            StandardButton {
-                id: exportKeyImagesButton
-                text: qsTr("Export key images") + translationManager.emptyString
-                small: true
-                visible: !appWindow.viewOnly
-                enabled: pageRoot.enabled
-                onClicked: {
-                    console.log("Transfer: export key images clicked")
-                    exportKeyImagesDialog.open();
-                }
-            }
-
-            StandardButton {
-                id: importKeyImagesButton
-                text: qsTr("Import key images") + translationManager.emptyString
-                small: true
-                enabled: appWindow.viewOnly && appWindow.isTrustedDaemon()
-                onClicked: {
-                    console.log("Transfer: import key images clicked")
-                    importKeyImagesDialog.open();
-                }
-            }
+            title: qsTr("Outputs") + translationManager.emptyString
+            button1.text: qsTr("Export") + translationManager.emptyString
+            button1.enabled: appWindow.viewOnly
+            button1.onClicked: console.log("Transfer: export outputs clicked")
+            button2.text: qsTr("Import") + translationManager.emptyString
+            button2.enabled: !appWindow.viewOnly
+            button2.onClicked: console.log("Transfer: import outputs clicked")
+            helpTextLarge1.text: qsTr("Required for cold wallets to sign their corresponding key images") + translationManager.emptyString
+            helpTextSmall1.text: qsTr("1. Using view-only wallet, export the outputs into a file") + translationManager.emptyString
+            helpTextSmall2.text: qsTr("2. Using cold wallet, import the outputs file") + translationManager.emptyString
+        }     
+
+        AdvancedOptionsItem {
+            visible: persistentSettings.transferShowAdvanced && appWindow.walletMode >= 2
+            title: qsTr("Key images") + translationManager.emptyString
+            button1.text: qsTr("Export") + translationManager.emptyString
+            button1.enabled: !appWindow.viewOnly
+            button1.onClicked: {
```

Comment:
Looks interesting but please use the following indentation to be inline with other code:
```
button1.onClicked: {
    console.log(...)
}
```